cargo-process.el

```diff
@@ -64,6 +64,10 @@
   "Set RUST_BACKTRACE environment variable to 1 for tasks test and run"
   :group 'cargo-process)
 
+(defcustom cargo-process--command-flags (list "--color never")
```

Comment:
From my comment below, this would just be a string. However I'm not sure why you're having this issue, maybe a specific Emacs client or Emacs configuration. So it would be best to have this as an empty string, to prevent this from breaking other people's workflow. You can then adjust the variable in your own config.